pages/sending_data.rst

```diff
@@ -90,6 +90,19 @@ Try sending an example message using curl::
 Both keep-alive and compression are supported via the common HTTP headers. The server will return a ``202 Accepted`` when the message
 was accepted for processing.
 
+Use Kafka as transport queue for sending
+========================================
+
+`Apache Kafka <http://kafka.apache.org>`__ can be used as a transport for different inputs in Graylog. Gelf, Syslog an RAW Inputs can be connected to a `Zookeeper <https://zookeeper.apache.org>`__ that controlls the Kafka Servers. The Topic is filtered by regex and depending on the Input you are able to set various additional settings.
+
+Examples can be found in the `Graylog Marketplace <https://marketplace.graylog.org/addons?search=kafka>`__.
```

Comment:
for amqp I would link to syslog via amqp as we do not have such a thing for kafka, i would need to create one ;)